packages/ibm-products-web-components/examples/create-side-panel/src/create-side-panel-stepped.ts

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
I may be confused by the name of the example/story here, I think this example is the same as the default... is it supposed to be multi-stepped?